hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/TestWriteBase.java

```diff
@@ -353,6 +361,21 @@ private void checkWrittenDataMor(File baseFile, Map<String, String> expected, in
       TestData.checkWrittenDataMOR(fs, latestInstant, baseFile, expected, partitions, schema);
     }
 
+    public TestHarness checkWrittenDataMergedMor(
+            ResolvedSchema schema,
+            String expected) throws Exception {
+      HoodieTableSource tableSource = new HoodieTableSource(
+              schema,
+              new Path(this.basePath),
+              Arrays.asList(conf.getString(FlinkOptions.PARTITION_PATH_FIELD).split(",")),
```

Comment:
Can we avoid to introduce `HoodieTableSource` here, the existing tool `TestWriteBase#checkWrittenData` does not work here ? If not, can we fix that method instead ?

Reply:
`TestWriteBase#checkWrittenData` can only reads the log file or only reads the parquet file, And the purpose of this `test case` is to test whether `MergeOnReadInputFormat` can sort and de duplicate as expected.  That's what I think.

Reply:
👌🏻 , I'm trying.

Reply:
resolved~